core/src/main/java/cz/xtf/core/openshift/HelmClients.java

```diff
@@ -0,0 +1,108 @@
+package cz.xtf.core.openshift;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.nio.file.Paths;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang3.SystemUtils;
+
+import cz.xtf.core.config.OpenShiftConfig;
+import cz.xtf.core.http.Https;
+
+public class HelmClients {
+
+    private static final String HELM_CLIENTS_URL = "https://mirror.openshift.com/pub/openshift-v4/clients/helm/latest";
```

Comment:
I believe we would need `latest` to be parametrized. `latest` is good default, but like we talked before there could be situation like OCP EUS, which will need we choose different version.

Reply:
Fixed with e82acc779300f8a3c2130f7e031820ea6c71d9b5